RecoTracker/MkFitCore/interface/TrackerInfo.h

```diff
@@ -24,17 +21,37 @@ namespace mkfit {
 
   //==============================================================================
 
+  struct ModuleInfo {
+    SVector3 m_pos;
+    SVector3 m_zdir;
+    SVector3 m_xdir;
+    unsigned int m_detid;
```

Comment:
```suggestion
    unsigned int detid;
```
since it's a POD public data, I don't think that `m_` is really necessary; but OK to keep

Reply:
replaced